src/ChangeFeedIterator.ts

```diff
@@ -0,0 +1,116 @@
+/// <reference lib="esnext.asynciterable" />
+import { isNumber } from "util";
```

Comment:
I wouldn't block this PR for it but removing all uses of `util` is on my radar for decreasing the bundle size

Reply:
ACK, but I'm gonna leave this as-is since I don't want to implement and write all those tests this week. When it comes time for you to remove this, you can ask me to do that, if you'd like.